cms/djangoapps/contentstore/utils.py

```diff
@@ -1159,7 +1165,9 @@ def load_services_for_studio(runtime, user):
         "settings": SettingsService(),
         "lti-configuration": ConfigurationService(CourseAllowPIISharingInLTIFlag),
         "teams_configuration": TeamsConfigurationService(),
-        "library_tools": LibraryToolsService(modulestore(), user.id)
+        "library_tools": LibraryToolsService(modulestore(), user.id),
+        "show_answer": ShowAnswerService(),
+        "show_correctness": ShowCorrectnessService(),
```

Comment:
This is only used in the Studio editor. We don't need it there.

Reply:
You can still submit problems in the Studio interface, so I think this code would still be invoked there.